src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;
```

Comment:
The variable name 'thirdLeg' is misleading. Since this is an isosceles trapezoid, it should have a top base, bottom base, and the same length legs. Consider renaming 'thirdLeg' to 'height'.